src/Behat/Behat/Tester/Cli/RerunController.php

```diff
@@ -74,6 +74,9 @@ public function configure(Command $command)
         $command->addOption('--rerun', null, InputOption::VALUE_NONE,
             'Re-run scenarios that failed during last execution.'
```

Comment:
Please could you change the existing `--rerun` help text to "Re-run scenarios that failed during last execution, or run everything if there were no failures"